src/app/api/comment/comment.controller.ts

```diff
@@ -0,0 +1,39 @@
+import { NextRequest, NextResponse } from 'next/server'
+import authenticate from '../core/utils/authenticate'
```

Comment:
please fix imports, we follow absolute import as convention

Reply:
Updated in 8315c817d2e398c277c36edbadab25a14b0e42fd commit